src/Resources/contao/controllers/FrontendIndex.php

```diff
@@ -69,8 +67,15 @@ public function run()
 			throw new PageNotFoundException('Page not found: ' . \Environment::get('uri'));
 		}
 
-		// Get the current page object(s)
-		$objPage = \PageModel::findPublishedByIdOrAlias($pageId);
+		return $this->renderPage(\PageModel::findPublishedByIdOrAlias($pageId));
+	}
+
+
+	public function renderPage($pageModel = null)
```

Comment:
It seems you need to also pass `$pageId` and `$objRootPage`, which are both used in the `renderPage()` method.